@commitlint/load/src/load.ts

```diff
@@ -84,6 +85,10 @@ export default async function load(
 			resolveFrom.silent(base, config.formatter) || config.formatter;
 	}
 
+	// Read plugins from extends
+	if (Array.isArray(extended.plugins))
+		config.plugins = union(config.plugins, extended.plugins || []);
```

Comment:
This is a bit more our code style 😄 

```suggestion
	if (Array.isArray(extended.plugins)) {
		config.plugins = union(config.plugins, extended.plugins || []);
	}
```